scripts/gen-universe.py

```diff
@@ -409,17 +411,12 @@ def write_package_in_zip(zip_file, path, package):
     :rtype: None
     """
 
-    package = copy.deepcopy(package)
+    package = downgrade_package_to_v2(package)
+
     package.pop('releaseVersion')
```

Comment:
The release version is not part of the package schema. Removing releaseVersion is just part of the conversion to the zip folder structure.